CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,90 @@
+
+### Reporting
+
+If you are being harassed by a member of our community, notice that someone else is being harassed, or have any other concerns, please contact the administrators via [codeofconduct@graphql.com](mailto:codeofconduct@graphql.com). If the person who is harassing you is on the admin team, they will not be involved in handling or resolving the incident.
+
+The admin team will respond to any complaint as promptly as possible we can. If you do not get a timely response (for example, if no admins are currently online) then please put your personal safety and well-being first, and consider logging out and/or contacting the admins by email at [codeofconduct@graphql.com](mailto:codeofconduct@graphql.com).
+
+This code of conduct applies to our community's spaces, but if you are being harassed by a member of our community outside our spaces, we still want to know about it. We will take all good-faith reports of harassment by our members, especially the administrators, seriously. This includes harassment outside our spaces and harassment that took place at any point in time. The abuse team reserves the right to exclude people from the community based on their past behavior, including behavior outside of our spaces and behavior towards people who are not in this community.
+
+In order to protect volunteers from abuse and burnout, we reserve the right to reject any report we believe to have been made in bad faith. Reports intended to silence legitimate criticism may be deleted without response.
+
+### Enforcement Process
+
+Every code of conduct violation report will be treated with seriousness and care. If a member's immediate safety and security is threatened, an individual admin may take any action that they deem appropriate, up to and including temporarily banning the offender from the community. In less urgent situations, at least two admins will discuss the offense and mutually arrive at a suitable response, which will be shared with the offender privately. Whatever the resolution that they decide upon, the decision of the admins involved in a violation case will be considered final.
+
+We will respect confidentiality requests for the purpose of protecting victims of abuse. At our discretion, we may publicly name a person about whom we've received harassment complaints, or privately warn third parties about them, if we believe that doing so will increase the safety of our members or the general public. We will not name harassment victims without their affirmative consent.
+
+GraphQL Summit prioritizes marginalized people’s safety over privileged people’s comfort. The administrators will not act on complaints regarding:
+
+- “Reverse”-isms, including “reverse racism,” “reverse sexism,” and “cisphobia”
+- Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”
+- Communicating in a “tone” you don’t find congenial
```

Comment:
Fix magic smart quotes (which are inconsistent with this docs usage otherwise).

```suggestion
- “Reverse”-isms, including "reverse racism," "reverse sexism," and "cisphobia"
- Reasonable communication of boundaries, such as "leave me alone," "go away," or "I’m not discussing this with you."
- Communicating in a "tone" you don’t find congenial
```